<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1034</width>
    <height>628</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(60, 174, 163);</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLCDNumber" name="hourLCD">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>141</width>
      <height>121</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(225, 225, 225);
background-color: rgb(23, 63, 95);</string>
    </property>
    <property name="digitCount">
     <number>2</number>
    </property>
    <property name="intValue" stdset="0">
     <number>22</number>
    </property>
   </widget>
   <widget class="QLCDNumber" name="minLCD">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>60</y>
      <width>141</width>
      <height>121</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(225, 225, 225);
background-color: rgb(23, 63, 95);</string>
    </property>
    <property name="digitCount">
     <number>2</number>
    </property>
    <property name="intValue" stdset="0">
     <number>22</number>
    </property>
   </widget>
   <widget class="QLCDNumber" name="secLCD">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>60</y>
      <width>141</width>
      <height>121</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(225, 225, 225);
background-color: rgb(23, 63, 95);</string>
    </property>
    <property name="digitCount">
     <number>2</number>
    </property>
    <property name="intValue" stdset="0">
     <number>22</number>
    </property>
   </widget>
   <widget class="QPushButton" name="imageDownloadButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(23, 63, 95);
color: rgb(225, 225, 225);</string>
    </property>
    <property name="text">
     <string>Image Download</string>
    </property>
   </widget>
   <widget class="QLabel" name="imageLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>250</y>
      <width>291</width>
      <height>191</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="zipCodeEdit">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>190</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(23, 63, 95);
color: rgb(225, 225, 225);</string>
    </property>
    <property name="text">
     <string>98119</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="weatherDownloadButton">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>190</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(23, 63, 95);
color: rgb(225, 225, 225);</string>
    </property>
    <property name="text">
     <string>Weather Download</string>
    </property>
   </widget>
   <widget class="QLabel" name="weatherLabel">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>250</y>
      <width>141</width>
      <height>191</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(225, 225, 225);
background-color: rgb(23, 63, 95);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignHCenter|Qt::AlignTop</set>
    </property>
   </widget>
   <widget class="QLabel" name="Greeting">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>441</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(225, 225, 225);
background-color: rgb(23, 63, 95);</string>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="DigitalPhotoFrame">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>10</y>
      <width>521</width>
      <height>431</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="weatherIcon">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>360</y>
      <width>141</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(23, 63, 95);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1034</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
